client/src/Dashboard/DashboardTitle.js

```diff
@@ -164,7 +164,7 @@ export default function DashboardTitle({ dates, setDates, makeMonth }) {
             className="record-payment-button"
             onClick={addPayment}
           >
-            {t('recordPaymentOf')} ${totalPayment.toFixed()}
+            {t('recordPaymentOf')} ${totalPayment}
```

Comment:
Will all the calculations initially be up the 2nd decimal point? Otherwise, we would need to round her for the initial load?

Reply:
@katelovescode